### ISSUE TITLE
Deployment of open accessibility website fails

### ISSUE THREAD

**Issue by UJehle** [goat-community/goat-website#101]

**Comment by UJehle** [comment]
I am setting this on "paused", as there are currently some unfinished and untested edits that were pushed to main (https://github.com/goat-community/goat-website/issues/96 & https://github.com/goat-community/goat-website/issues/95). We first need to finalize those before building the website again.
